README.md

```diff
@@ -440,8 +452,7 @@ To enable the `prometheus` plugin in Kong:
 
      Alternatively, use the Admin API to list plugins for a given Service:
      ```shell
-     PLUGINS_URL="http://<CONTROL_PLANE_HOST>:8001/services/<SERVICE_NAME>/plugins"
-     curl -H "Kong-Admin-Token: $KONG_ADMIN_TOKEN" ${PLUGINS_URL}
+     curl http://localhost:8001/plugins
```

Comment:
The plugins are applied to the global scope by defalut, accessing plugins on service level will result null response. So correcting it.